modules/installation-guide/examples/checluster-properties.adoc

```diff
@@ -45,125 +45,127 @@ pluginRegistryUrl: Public URL of the plugin registry that serves sample ready-to
 
 [cols="2,5", options="header"]
-:===
- Property: Description
-externalIdentityProvider: Instructs the Operator on whether to deploy a dedicated {platforms-identity-provider} instance. By default, a dedicated {platforms-identity-provider} server is deployed as part of the {prod-short} installation. When `externalIdentityProvider` is `true`, no dedicated {platforms-identity-provider} will be deployed by the Operator and you will need to provide details about the external {platforms-identity-provider} you are about to use. See also all the other fields starting with\: `identityProvider`.
-identityProviderAdminUserName: Overrides the name of the {platforms-identity-provider} administrator user. Defaults to `admin`.
-identityProviderClientId: Name of a {platforms-identity-provider}, `client-id` that is used for {prod-short}. Override this when an external {platforms-identity-provider} is in use. See the `externalIdentityProvider` field. When omitted or left blank, it is set to the value of the `flavour` field suffixed with `-public`.
-identityProviderContainerResources: {platforms-identity-provider} container custom settings.
-identityProviderImage: Overrides the container image used in the {platforms-identity-provider} deployment. This includes the image tag. Omit it or leave it empty to use the default container image provided by the Operator.
-identityProviderImagePullPolicy: Overrides the image pull policy used in the {platforms-identity-provider}, deployment. Default value is `Always` for `nightly` or `latest` images, and `IfNotPresent` in other cases.
+:=== 
+ Property: Description 
+externalIdentityProvider: Instructs the Operator on whether or not to deploy a dedicated Identity Provider (Keycloak or RH SSO instance). Instructs the Operator on whether to deploy a dedicated Identity Provider (Keycloak or RH-SSO instance). By default, a dedicated Identity Provider server is deployed as part of the Che installation. When `externalIdentityProvider` is `true`, no dedicated identity provider will be deployed by the Operator and you will need to provide details about the external identity provider you are about to use. See also all the other fields starting with\: `identityProvider`.
+identityProviderAdminUserName: Overrides the name of the Identity Provider administrator user. Defaults to `admin`.
+identityProviderClientId: Name of a Identity provider, Keycloak or RH-SSO, `client-id` that is used for Che. Override this when an external Identity Provider is in use. See the `externalIdentityProvider` field. When omitted or left blank, it is set to the value of the `flavour` field suffixed with `-public`.
+identityProviderContainerResources: Identity provider container custom settings.
+identityProviderImage: Overrides the container image used in the Identity Provider, Keycloak or RH-SSO, deployment. This includes the image tag. Omit it or leave it empty to use the default container image provided by the Operator.
```

Comment:
Why is this `Identity Provider, Keycloak or RH-SSO,` when 4 lines earlier you have `Identity Provider (Keycloak or RH SSO instance)` ?